AGENTS.md

```
@@ -0,0 +1,169 @@
+### API User Secrets (Required)
+
+```bash
+cd api/MMRProject.Api
+dotnet user-secrets set "Authorization:Issuer" "<clerk_issuer_url>"
+```
+
+The Clerk issuer URL is found in Clerk Dashboard under API Keys (e.g., `https://your-app.clerk.accounts.dev`)
+
+### MMR API (.env)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)